server/recceiver/cfstore.py

```diff
@@ -124,6 +124,7 @@ def _startServiceWithLock(self):
                     self.client.set(property={"name": cf_property, "owner": owner})
 
                 self.record_property_names_list = set(record_property_names_list)
+                self.managed_properties = required_properties.union(set(record_property_names_list))
```

Comment:
Do you need the `set` here? I think you can union any iterable... 🔧

Reply:
You could also use the union operator `|` 🔧

Reply:
fixed